services/QuillLMS/client/app/bundles/Shared/components/shared/dropdownInput.tsx

```diff
@@ -47,6 +47,7 @@ interface DropdownInputState {
 
 const KEYDOWN = 'keydown'
 const MOUSEDOWN = 'mousedown'
+const TOUCHSTART = 'touchstart'
```

Comment:
I assume that this is an accessibility enhancement?

Reply:
@anathomical responsive design! I swear it used to work, but for whatever reason the `dropdownInput` stopped responding to touchevents on mobile.

Reply:
Probably some update to the APIs that they forgot to tell everyone about before making.